RNTester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -19,7 +19,6 @@
 		5C60EB1C226440DB0018C04F /* AppDelegate.mm in Sources */ = {isa = PBXBuildFile; fileRef = 5C60EB1B226440DB0018C04F /* AppDelegate.mm */; };
 		5CB07C9B226467E60039471C /* RNTesterTurboModuleProvider.mm in Sources */ = {isa = PBXBuildFile; fileRef = 5CB07C99226467E60039471C /* RNTesterTurboModuleProvider.mm */; };
 		68E1E4BC2230DF2F00570185 /* ComponentRegistry.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 68E1E4BB2230DF2F00570185 /* ComponentRegistry.cpp */; };
-		7DBD3B12498EA276DB551187 /* libPods-RNTester-macOS.a in Frameworks */ = {isa = PBXBuildFile; fileRef = B334FFB3ADFDA3CD1AC56276 /* libPods-RNTester-macOS.a */; };
```

Comment:
This will be added back once we add an actual macOS target to the pods version of RNTester.